src/utils/ComponentsTypes.ts

```diff
@@ -17,7 +16,7 @@ export class ComponentsTypes {
   }
 
   public static get allFacetsClassname() {
-    return ComponentsTypes.allFacetsType.map(type => Component.computeCssClassNameForType(type));
+    return ComponentsTypes.allFacetsType.map(type => `Coveo${type}`);
```

Comment:
Is it better to do this than use `computeCssClassNameForType`? I may have been doing this wrong.

Reply:
It's "better", but in this case it was to break a circular dependency, since it is a module that is also imported by `Component`